packages/new-polymath-issuer/src/pages/DividendsWizard/Step-2/TaxWithholdingModal.tsx

```diff
@@ -52,34 +54,45 @@ export const TaxWithholdingModal: FC<Props> = ({
 
     const valueAddress = value[csvEthAddressKey].toUpperCase();
 
-    const matchingIndex = findIndex(
-      formTaxWithholdings,
-      taxWithholding =>
-        taxWithholding[csvEthAddressKey].toUpperCase() === valueAddress
-    );
+    // Check if excluded
+    const excluded = exclusionList.find(address => {
+      return address === valueAddress;
+    });
 
-    const alreadyExists = matchingIndex !== -1;
+    if (!excluded) {
```

Comment:
This whole section could use some comments explaining what is being done. I had a bit of a hard time trying to understand the code